src/components/notifications/AppExplorer/AppCard/index.tsx

```diff
@@ -108,7 +109,7 @@ const AppCard: React.FC<AppCardProps> = ({ name, description, logo, bgColor, url
           <Text className="" variant="large-600">
             {name}
           </Text>
-          <VerifiedIcon />
+          {isFeatured? <VerifiedIcon /> : null}
```

Comment:
```suggestion
          {isFeatured ? <VerifiedIcon /> : null}
```

Reply:
I'm a bit confused by this, shouldn't the verified badge be based on `is_verified/isVerified` in the API response not whether the project is featured or not?

For GM dapp I'm seeing `/notify-config` return:

```
{
    "id": "15b8521b1744afdca76e84668691f7207ee1d578b49feb7bc1d66382974a1678",
    "name": "GM Dapp",
    "homepage": "https://gm.walletconnect.com",
    "image_id": "ed73e2f2-7837-463b-4026-c62007b1c000",
    "description": "Get a gm every hour",
    "dapp_url": "https://gm.walletconnect.com",
    "is_featured": true,
    "order": 10,
    "image_url": {...},
    "notificationTypes": [...],
    "isVerified": true
}
```